hazelcast/test/src/sql_test.cpp

```diff
@@ -196,6 +210,30 @@ class hz_serializer<test::portable_pojo> : public portable_serializer
 constexpr int32_t hz_serializer<test::portable_pojo>::PORTABLE_FACTORY_ID;
 constexpr int32_t hz_serializer<test::portable_pojo>::PORTABLE_VALUE_CLASS_ID;
 
+template<>
+struct hz_serializer<test::student> : public portable_serializer
```

Comment:
I saw it but i thought clash is not a problem because `portable_pojo_key` clashes already. So do you want me to move both of them to `Serializables.h` and make their ids unique ?